docs/reference/security.rst

```diff
@@ -339,22 +311,22 @@ and then configure SonataAdminBundle:
                 acl_user_manager: App\Manager\AclUserManager
                 # ...
 
-In your ``config/packages/fos_user.yaml`` you then need to put the following:
+In your ``config/packages/sonata_user.yaml`` you then need to put the following:
```

Comment:
I think this whole block can be removed

Reply:
Sorry I'm not sure what block you're talking about. I should remove line 314 to 325 (the `config/packages/sonata_user.yaml` code block) ?

Reply:
Yes, exactly! Can you do it?

Reply:
Np, I'll do it tomorrow or this week-end.